app/src/main/java/mozilla/org/webmaker/javascript/WebAppInterface.java

```diff
@@ -59,4 +66,10 @@ public void logText(String txt){
     public void setView(String url) {
         Router.sharedRouter().open(url);
     }
+
+    @JavascriptInterface
+    public String getRouteParams() {
+        Log.v("Router", mRoute.toString());
+        return mRoute.toString();
```

Comment:
I suspect this'll throw nullpointer exceptions at us at some time in the future when we forgot about this already again...
```
if (mRoute==null) {
  return "";
}
return mRoute.toString();
```